documentation/architecture/decisions/0003-crate-feature-flags.md

```diff
@@ -80,17 +116,24 @@ no\_std without alloc:
 
 To effect conditional compilation on `std`, `no_std+alloc` and `no_std` builds the following attributes can be used:
 
-    // only compile for std
-    #[cfg(feature = "std")]
+```rust
```

Comment:
This is a lot of changes from how we do things now, but *hopefully* it should make the common cases easier.